commands-extender/commands-provider.ts

```diff
@@ -2,27 +2,56 @@ import { CommandProvider, CommandsData, Command, COMMANDS_TOKEN, CommandsTarget,
 import { Observable } from "rxjs";
 import { ClassProvider, Injectable } from "@angular/core";
 import { Router } from "@angular/router";
+import { PrintPreviewCommand } from "./print-preview.command";
 
+/**
+ * The category name in which to place the custom commands.
```

Comment:
We can add that:

*Categories are used to group similar commands in the action menu*